src/Diagnostics.RuntimeHost/SerializationConvertors/AsRuntimeTypeConverter.cs

```diff
@@ -20,6 +22,8 @@ public class AsRuntimeTypeConverter<T> : JsonConverter<T>
             };
 
             options.Converters.Add(new ExceptionConverter());
+            options.Converters.Add(new DevOpsGetBranchesConverter());
+            options.Converters.Add(new DevOpsMakePRConverter());
```

Comment:
I don't think you need these lines. You will need them if the DevOps responses need to be handled with AsRuntimeTypeConverter. But looking at your changes in Startup.cs, it's not the case.
It won't be harmful to leave them here, however minimizing the number of Converter objects is essential for performance. Can you try removing them and see if it still works?